__tests__/registries/npm-registry.js

```diff
@@ -175,6 +175,24 @@ describe('request', () => {
 
     expect(requestParams.headers.authorization).toBe('Bearer testAuthToken');
   });
+
+  test('should add authorization header with correct token for scoped package', () => {
```

Comment:
This test case is very similar to the previous one so I'd either make them a bit more differentiated or fix the previous one.